test/test_edit_box.rb

```diff
@@ -0,0 +1,60 @@
+# frozen_string_literal: true
+
+require "test_helper"
+
+class TestEditBox < Minitest::Test
+  def setup
+    @app = Minitest::Mock.new
+    @app.expect :append, nil, [String]
+    @app.expect :bind, nil, [Object]
```

Comment:
Lot of mocking here. Not sure if this is the right approach as I'm less familiar with minitest (we use rspec at work). Let me know if there's a better way to do this

Reply:
There's a lot of ways to approach mocking and I'm one of the rare people who is open to most. Looks fine to me!